src/main/java/nextstep/courses/domain/Session/Session.java

```diff
@@ -6,50 +6,54 @@
 import java.util.List;
 
 public class Session {
-    private final Duration duration;
+    private final SessionDuration sessionDuration;
     private final Image coverImage;
     private final int fee;
     private final int maxStudents;
     private final Students students = new Students();
-    private Status status = Status.READY;
+    private SessionStatus sessionStatus = SessionStatus.READY;
 
     public Session(LocalDate startDate, LocalDate endDate, Image coverImage) {
-        this.duration = new Duration(startDate, endDate);
+        this.sessionDuration = new SessionDuration(startDate, endDate);
         this.coverImage = coverImage;
         this.fee = 0;
         this.maxStudents = Integer.MAX_VALUE;
     }
 
     public Session(LocalDate startDate, LocalDate endDate, Image coverImage, int fee, int maxStudents) {
-        this.duration = new Duration(startDate, endDate);
+        this.sessionDuration = new SessionDuration(startDate, endDate);
         this.coverImage = coverImage;
         this.fee = fee;
         this.maxStudents = maxStudents;
     }
 
     public void apply(List<Student> studentList) {
```

Comment:
```suggestion
    public Enrollment enrollment(List<Student> studentList) {
```
수강 신청을 위해 필요한 값을 가지는 Enrollment와 같은 객체를 추가하고 이 객체가 수강 신청 로직을 담당하도록 구현하는 것은 어떨까?
그런데 수강 신청을 하려면 현재 수강생 전체 목록과 새롭게 수강신청하려는 사용자 정보를 인자도 전달해야 하지 않을까?